route/build/cpl/RtmMod.F90

```diff
@@ -246,13 +247,13 @@ SUBROUTINE route_ini(rof_active,flood_active)
     ! 5. For continue and/or Branch run-- Initialize restart and history files
     !-------------------------------------------------------
     ! Obtain last restart name, and obtain and open last history file depending on which run types-- contiuous or branch
-    if ((nsrest == nsrContinue) .or. &
-        (nsrest == nsrBranch)) then
+    if ((nsrest == nsrContinue) then
       call RtmRestGetfile()
-      if (nsrest == nsrContinue) then
-        call init_histFile(ierr, cmessage)
-        if(ierr/=0)then; call shr_sys_abort(trim(subname)//trim(cmessage)); endif
-      end if
+      isColdStart=.false.
+      call init_histFile(ierr, cmessage)
+      if(ierr/=0)then; call shr_sys_abort(trim(subname)//trim(cmessage)); endif
+    else
```

Comment:
Actually in this case now, we should listen to what's on the control file and use what it's saying...

Reply:
So should I change if statement from `if (nsrest == nsrContinue) then` to `if (continue_run) then`? 

if i change now, it won't work because there is no mechanism to change continue_run in control file right now, so wait till this mechanism is implemented (in Buildnml?)

Reply:
I would leave the if statement as

`if (nsrest == nsrContinue) then`

as you point out we can't change it until the buildnml change happens. And even then you might as well leave it as above, there isn't enough of a reason to change it.

Reply:
My point was actually to now remove the

``` fortran
else
   isColdStart = .true.
```

because now we want that to set by the control file if it isn't a nsrContinue case.

Reply:
ok. I removed those two lines.